test/built-ins/decodeURI/throws-URIError.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2025 ayuan.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: |
+    Verify decodeURIComponent throws URIError for various invalid UTF-8 sequences
+    Reference: https://stackoverflow.com/a/1319229/172999
+esid: sec-decodeuricomponent-encodeduricomponent
+description: |
+    Invalid sequences include:
+    - Surrogate pair encoding
+    - Overlong encoding
+    - Invalid continuation bytes
+    - Incomplete sequences
+    - Out-of-range code points
```

Comment:
Could you rearrange the `info` and `description` a bit? The `description` is supposed to fit in one, or at most two, lines as a summary, and the `info` can be a long description with links.
```suggestion
description: Verify decodeURIComponent throws URIError for various invalid UTF-8 sequences
esid: sec-decodeuricomponent-encodeduricomponent
info: |
    Invalid sequences include:
    - Surrogate pair encoding
    - Overlong encoding
    - Invalid continuation bytes
    - Incomplete sequences
    - Out-of-range code points
    Reference: https://stackoverflow.com/a/1319229/172999
```